specification/transport/abstract.tex

```diff
@@ -693,13 +696,30 @@ \subsubsection{Behaviors}
 any other successfully reassembled transfer under the same session specifier by more than the transfer-ID timeout
 is considered unique regardless of its transfer-ID value.
 
+If the optimal transfer-ID timeout value for a given session cannot be known in advance,
+it can be computed at runtime as a multiple of the average transfer reception interval (on this session).
```

Comment:
Let's decline to comment on any specific algorithm in the normative part of the specification. I think there's more work one would need to do for this to be better then a hard-coded default especially given a per-session limit and for any system where most multi-part transfers are 2 or 3 frames.

Reply:
Moved into a footnote.